enzyme/tests/TransactionCreate.test.tsx

```diff
@@ -64,38 +62,77 @@ describe('TransactionCreate component', () => {
 
     it('should render the expected snapshot', async () => {
         const component: any = renderer
-            .create(<TransactionCreate activeSmartContract={greenContract}/>)
+            .create(<TransactionCreate activeSmartContract={greenContract} postMessageHandler={postMessageHandlerStub}/>)
             .toJSON();
         expect(component).toMatchSnapshot();
     });
 
     it('redirects back to the home page when the appropriate link is clicked on', async () => {
-        const component: any = mount(<TransactionCreate activeSmartContract={greenContract}/>);
+        const component: any = mount(<TransactionCreate activeSmartContract={greenContract} postMessageHandler={postMessageHandlerStub}/>);
         component.find('.titles-container > span').simulate('click');
         changeRouteStub.should.have.been.called;
     });
 
     it('generates transaction arguments when an option from the transaction select is chosen', async () => {
-        const component: any = mount(<TransactionCreate activeSmartContract={greenContract}/>);
+        const component: any = mount(<TransactionCreate activeSmartContract={greenContract} postMessageHandler={postMessageHandlerStub}/>);
         expect(component.state().transactionArguments).toBe('');
-        component.find('select').at(0).prop('onChange')( { currentTarget: { value: 'transactionOne' } });
+        component.find('select').at(0).prop('onChange')( { currentTarget: { value: 'transactionOne' } } );
         getTransactionArgumentsSpy.should.have.been.called;
-        expect(component.state().transactionArguments).toBe('name: \n');
+        expect(component.state().activeTransaction).toBe(transactionOne);
```

Comment:
I've only just noticed this - I think we should be using the `should` syntax like we do everywhere else instead of `expect`.